src/catalog/catalog.cpp

```diff
@@ -283,6 +283,19 @@ CatalogEntryLookup Catalog::LookupEntry(ClientContext &context, CatalogType type
 	return {nullptr, nullptr};
 }
 
+CatalogEntry *Catalog::GetEntry(ClientContext &context, const string &schema, const string &name) {
+	vector<CatalogType> entry_types {CatalogType::TABLE_ENTRY, CatalogType::SEQUENCE_ENTRY};
+
+	for (auto entry_type : entry_types) {
+		CatalogEntry *result = GetEntry(context, entry_type, schema, name, true);
+		if (result != nullptr) {
+			return result;
+		}
+	}
+
+	throw CatalogException("CatalogElement " + schema + "." + name + "does not exist!");
```

Comment:
Missing space here, should be `" does not exist!"`

Or better yet, use printf formatting:

```cpp
	throw CatalogException("CatalogElement \"%s.%s\" does not exist!", schema, name);
```